src/janitor.c

```diff
@@ -27,6 +27,7 @@
 /* do full maintenance 3x per second */
 static struct timeval full_maint_period = {0, USEC / 3};
 static struct event full_maint_ev;
+extern bool any_user_level_timeout_set;
```

Comment:
Let's rename this variable, because its name is too generic now that #1189 is merged.
```suggestion
extern bool any_user_level_server_timeout_set;
```